config/logging.yaml

```diff
@@ -62,3 +67,4 @@
 :test:
   :level: debug
   :filename: "test.log"
+  :truncate: true
```

Comment:
*THANK YOU* :-) I guess I can change my alias to run rails server:

    alias bes='[ -x bin/smart-proxy ] && bundle exec bin/smart-proxy || rm log/*log; tsocks bundle exec rails s -b 0.0.0.0'